src/razor/src/blazorDebug/blazorDebugConfigurationProvider.ts

```diff
@@ -125,6 +184,13 @@ export class BlazorDebugConfigurationProvider implements vscode.DebugConfigurati
         inspectUri: string,
         url: string
     ) {
+        const originalInspectUri = inspectUri;
+        const useVSDbg = await BlazorDebugConfigurationProvider.useVSDbg('');
```

Comment:
Should this try to workout the path to the project using `configuration.cwd` if it is set like [here](https://github.com/thaystg/vscode-csharp/blob/4e7091703de13a06ec5abb2eb61fcafcbe24fea2/src/razor/src/blazorDebug/blazorDebugConfigurationProvider.ts#L102)?

Reply:
done.